docs/examples/cookbook/drag-and-drop.md

```diff
@@ -99,7 +99,7 @@ local function Draggable(
 		end),
 		-- Calculated manually so the Scale can be set relative to
 		-- `props.Parent` at all times, rather than the `Parent` of this Frame.
-		Size = scope:Computed(function(use)
+		Size = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		Size = scope:Computed(function(use)
```

Reply:
reverted